lib/hooks/create/place.js

```diff
@@ -1,9 +1,39 @@
-const { get } = require('lodash');
+const { get, omit } = require('lodash');
+const { diff } = require('deep-diff');
 const { autoResolved, withInspectorate, withLicensing } = require('../../flow/status');
 
+const normalize = (path, key, lhs, rhs) => {
+  // ignore array order
+  if (key === 'holding' || key === 'suitability') {
+    Array.isArray(lhs) && lhs.sort();
+    Array.isArray(rhs) && rhs.sort();
+    return [lhs, rhs];
+  }
+
+  // handle different falsy values (null, empty string, undefined)
+  if (!lhs && !rhs) {
+    return [undefined, undefined];
+  }
+
+  return false;
+};
+
+const onlyRolesChanged = model => {
+  const currentModel = get(model, 'data.modelData');
+  const data = get(model, 'data.data');
+
+  const oldValues = omit(currentModel, ['id', 'deleted', 'createdAt', 'updatedAt', 'nacwoId', 'migratedId']);
+  const newValues = omit(data, ['roles']);
+  const changes = diff(oldValues, newValues, { normalize });
+
+  return !changes;
```

Comment:
whoa.gif

Reply:
I didn't want to hard code the fields that we compare in case we add new fields to the place table.